src/pages/organize/[orgId]/campaigns/calendar/index.tsx

```diff
@@ -71,9 +71,13 @@ const AllCampaignsCalendarPage : PageWithLayout<AllCampaignsCalendarPageProps> =
         setCalendarView(event.target.value as string);
     };
 
+    const sortedEvents = [...events].sort((a, b) => {
+        return new Date(a.start_time).getTime() - new Date(b.start_time).getTime();
```

Comment:
I will leave this as is for the time being -- directly subtracting dates raises a TS error since the lefthand operand can't be of type Date. Workarounds recommend using typecasting date as any or using valueOf() of getTime(). If you have a better idea I will implement!

Reply:
I did not know that! Sounds like a flaw in TS, but I don't mind keeping it this way. 😊